apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -48,7 +43,7 @@ function renderSectionProjects(sectionId) {
       <SectionProjectsList
         projectsData={projectsData}
         studioUrlPrefix={studioUrlPrefix}
-        showProjectThumbnails={showProjectThumbnails}
```

Comment:
With this change, do we ever plan to use `SectionProjectsList` where `showProjectThumbnails={false}` ? I'm wondering if the next step is to pull out this prop entirely?

Reply:
Good question.